recipes/deepspeech/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "deepspeech" %}
+{% set version = "0.9.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/mozilla/DeepSpeech/archive/v{{ version }}.tar.gz
+  sha256: a8e731a27a7cac4b4d0ae72b8e91493a5a372888dca10a11ece65ce16fb9b1b2
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [win]
```

Comment:
Why are you skipping Windows? Can this recipe be `noarch`?

Reply:
There's a dependency that doesn't have windows build available

Reply:
In those cases we make the recipe `noarch` and just ignore the failing build on windows here when we merge (noarch builds are only done on Linux in the feedstock)